### DISCUSSION TITLE
Interactive photo frame/toy ili9488

### DISCUSSION THREAD

**Discussion by zalves** [micropython/micropython#14107] [Ideas]
I want to make a box with 10 buttons that when they are pressed they display a certain image. I've managed with an ili9341 display, but I have a bigger 3.5 inch ili9488 that I would prefer to use. but so far haven't been able to find a way to use this display with MicroPython. 
My end goal is to make a toy for my daughter with numbers, animals and some other things to play with.

For the ili9341 I used this driver that works very well, I can use a command such as this: 
"display.draw_image('images/MicroPython128x128.raw', 0, 129, 128, 128)" to display an image.
https://github.com/rdagger/micropython-ili9341

But for the ili9488 can't find anything that works similarly. Is this display not usable with MicroPython?

**Comment by robert-hh** [comment]
For larger displays the SSD1963 or RA8875 are more common. I used a display with a SSD1963 a while ago for a display. LCD modules are available in sizes between 4.3" and 9". Same for RA8875. For all kind of drivers you may look at the repository of mcauser or in https://awesome-micropython.com/, which aggregates all kind of drivers.
Edit: A good source for all display kinds is [www.buydisplay.com](https://www.buydisplay.com/)

**Reply by zalves** [reply]
Thank you, makes it easier to choose hardware knowing what libraries are available. Have a great day

**Reply by peterhinch** [reply]
[This driver](https://github.com/peterhinch/micropython-nano-gui/blob/master/drivers/ili94xx/ili9486.py) is for the 9486 but is likely to work as it uses a minimal set of common features.

**Reply by zalves** [reply]
I'm not a programmer and it's been hard to understand how all works. Don't have much time but have been learing, and I'm in love with micropython, easier to grasp than C and no compilation all the time. 

I'm impressed how there is such big comunity devoloping all sorts of libraries and firmwares. Hope to learn enough so one day I'll be able to contribute.
